README.md

```diff
@@ -43,8 +43,12 @@ end
 
 ```ruby
 class Song < ActiveRecord::Base
-  include Metka::Model(column: 'tags')
-  include Metka::Model(column: 'genres')
+  include Metka::Model
```

Comment:
Maybe we could include `Metka::Model` to the `ActiveRecord::Base` from the gem, so we could just use a single line of code in the model (in the case with a single metka-column involved)?

```
has_metka { column :tags }
```